packages/gestalt/src/Icon.js

```diff
@@ -76,7 +78,7 @@ export default function Icon(props: Props) {
       aria-label={accessibilityLabel}
       role="img"
     >
-      <title>{accessibilityLabel}</title>
+      <title>{accessibilityLabel || title}</title>
```

Comment:
should this be the opposite now? `title || accessibilityLabel`?

Reply:
If we're exposing `title` to users (as indicated in your other comment), then yep, I think that makes sense.